partition/bootloader.go

```diff
@@ -146,3 +155,13 @@ func MarkBootSuccessful(bootloader Bootloader) error {
 
 	return bootloader.SetBootVars(m)
 }
+
+func reboot() error {
```

Comment:
I thought about that, but how do you unit-test something that reboots the system (or fails if not root)?...